src/main/java/taxi/service/DriverServiceImpl.java

```diff
@@ -38,6 +39,7 @@ public boolean delete(Long id) {
 
     @Override
     public Driver findByLogin(String login) {
-        return driverDao.findByLogin(login).get();
+        return driverDao.findByLogin(login)
+                .orElseThrow(() -> new DataProcessingException("No such login " + login));
     }
```

Comment:
did you check it?
it will not be visible for user in browser

Reply:
year, server give 500 error

Reply:
![image](https://user-images.githubusercontent.com/112757586/204331691-bbe00c5d-02cf-43ed-a754-028b041b03bb.png)

Reply:
i add red msg on top
![image](https://user-images.githubusercontent.com/112757586/204335835-55c17592-3c49-49e1-a791-210cba9ec911.png)

Reply:
![image](https://user-images.githubusercontent.com/112757586/204338906-d197be99-56c6-43a9-b294-d96d539ced46.png)